functional_tests/step_helpers/release_page_helpers.py

```diff
@@ -0,0 +1,287 @@
+    context.page.locator(locator).fill(text)
+
+
+# 'index' parameter is used when inserting blocks because when multiple blocks are inserted,
+# button may shift causing inconsistent increments in the block indices (e.g. add_multiple_descriptions)
+def insert_block_under_pre_release_access(context: Context, block_name: str, index: int = 0) -> None:
+    """Inserts new empty block under pre-release access."""
+    content_panel = context.page.locator("#panel-child-content-pre_release_access-content")
+
+    content_panel.get_by_role("button", name="Insert a block").nth(index).click()
+    context.page.get_by_role("option", name=block_name).click()
+
+
+def add_basic_table_block_under_pre_release_access(context: Context, data: bool = True, header: bool = True) -> None:
+    """Inserts a table block and fills with content."""
+    insert_block_under_pre_release_access(context, block_name="Basic table")
+    if header:
+        context.page.get_by_label("Table headers").select_option("column")
+    context.page.get_by_role("textbox", name="Table caption").fill("Caption")
+
+    if data:
+        context.page.locator("td").first.click()
+        context.page.keyboard.type("Name")
+        context.page.locator("td:nth-child(2)").first.click()
+        context.page.keyboard.type("Role")
+        context.page.locator("td").first.click()
+        context.page.keyboard.type("Heading")
+        context.page.locator("tr:nth-child(2) > td").first.click()
+        context.page.keyboard.type("Jack Smith")
+        context.page.locator("tr:nth-child(2) > td:nth-child(2)").first.click()
+        context.page.keyboard.type("Publisher")
+
+
+def add_description_block_under_pre_release_access(context: Context, index: int = 0) -> None:
+    """Inserts description and fills with text under pre-release access."""
+    insert_block_under_pre_release_access(context, block_name="Description", index=index)
+    context.page.get_by_role("region", name="Description *").get_by_role("textbox").fill("Description")
+
+
+def expect_text(
+    context: Context,
+    key: str | None,
+    texts: Sequence[str] | dict[str, str] | Mapping[str, str | Sequence[str]],
+) -> None:
+    if key:
+        texts = texts.get(key)
+    if isinstance(texts, str):
+        texts = [texts]
+    for text in texts:
+        expect(context.page.get_by_text(text)).to_be_visible()
+
+
+def add_date_change_log(context: Context) -> None:
+    change_to_release_date_section = context.page.locator("#panel-child-content-changes_to_release_date-section")
+    change_to_release_date_section.get_by_role("button", name="Insert a block").click()
+    change_to_release_date_section.get_by_label("Reason for change*").fill("Updated due to data availability")
+
+
+def add_release_date_text(context: Context) -> None:
+    fill_locator(context, RELEASE_DATE_TEXT, "March 2025 to August 2025")
+
+
+def add_next_release_date_text(context: Context) -> None:
```

Comment:
You can inline the function call from `add_next_release_date_text` directly into line 172. 

That means, instead of calling `add_next_release_date_text(context)`, you would replace it with the body of that function: `fill_locator(context, NEXT_RELEASE_DATE_TEXT, "To be confirmed")`

So, in line 172 (inside the FEATURE_ACTIONS dictionary), you can replace the value for the key "a next release date text" with a lambda that performs the same action

`"a next release date text": lambda context: fill_locator(context, NEXT_RELEASE_DATE_TEXT, "To be confirmed"),`